deptrack-devops-examples/app/DevVM.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TupleSections     #-}
+module Main where
```

Comment:
I'd be happy if you could modify the `README.md` to also plot one `local` and the `remote` graph. This example will serve as a second example of using `appMain`.

Longer term, if you want to add new binaries (if you use it for yourself) I'd recommend creating a repository like I did for `deptrack-builder` and `deptrack-devbox` so that newcomers can see "real projects" as well. We should link these examples too. That said, when we add new important features we should make sure to add examples in this directory as well (anyway we need some beefy example when "testing something").